src/API/useInAppUpload.js

```diff
@@ -2,17 +2,31 @@ import { useMutation } from 'react-query';
 
 import { useOkapiKy } from '@folio/stripes/core';
 
+import { CAPABILITIES_VALUE } from '../constants';
+
 export const useInAppUpload = () => {
   const ky = useOkapiKy();
 
-  const { mutateAsync: inAppUpload, isLoading } = useMutation({ mutationFn: ({ jobId, contentUpdates }) => {
-    return ky.post(`bulk-edit/${jobId}/item-content-update/upload`, {
-      searchParams: { limit: 10 },
-      json: {
-        itemContentUpdates: contentUpdates,
-        totalRecords: contentUpdates.length,
-      },
-    }).json();
+  const { mutateAsync: inAppUpload, isLoading } = useMutation({ mutationFn: ({ jobId, contentUpdates, capability }) => {
+      const typeOfBulk = CAPABILITIES_VALUE[capability];
+      const getBody = () => {
+            if (typeOfBulk === CAPABILITIES_VALUE.ITEMS) {
+                return {
+                    itemContentUpdate: contentUpdates,
+                    totalRecords: contentUpdates.length,
+                };
+            } else {
+                return {
+                userContentUpdate: contentUpdates,
+                totalRecords: contentUpdates.length,
+                };
+            }
+      };
+
+        return ky.post(`bulk-edit/${jobId}/${typeOfBulk}-content-update/upload`, {
+          searchParams: { limit: 10 },
+          json: getBody(),
```

Comment:
getBody looks strange :) from one point it encapsulate logic to prepare body, from another typeOfBulk is our of this function

Reply:
@NikitaSedyx you mean naming here? I should escape of `get` name because I use  param outside the func?

Reply:
I mean that you use typeOfBulk inside getBody only, but it's globally defined in mutation callback

Reply:
@NikitaSedyx , I use typeOfBulk not only in getBody, I also use it in our url